idp-default.properties

```diff
@@ -0,0 +1,264 @@
+# The error code will typically be login_required, interaction_required, or another code defined in Section 3.1.2.6.
+# This can be used as a method to check for existing authentication and/or consent.
+# login-The Authorization Server SHOULD prompt the End-User for reauthentication. If it cannot reauthenticate the End-User, \
+#  it MUST return an error, typically login_required.
+# consent-The Authorization Server SHOULD prompt the End-User for consent before returning information to the Client.
+# If it cannot obtain consent, it MUST return an error, typically consent_required.
+# select_account-The Authorization Server SHOULD prompt the End-User to select a user account. This enables an End-User
+# who has multiple accounts at the Authorization Server to select amongst the multiple accounts that they might have current
+# sessions for. If it cannot obtain an account selection choice made by the End-User, it MUST return an error,
+# typically account_selection_required.
+mosip.idp.supported.ui.prompts={'none','login','consent','select_account'}
+
+## Type of the client assertion
+mosip.idp.supported.client.assertion.types={'urn:ietf:params:oauth:client-assertion-type:jwt-bearer'}
+
+## Type of the client authentication methods for token endpoint
+mosip.idp.supported.client.auth.methods={'private_key_jwt'}
+
+## ---------------------------------------- Cache configuration -------------------------------------------
+mosip.idp.cache.secure.individual-id=true
+mosip.idp.cache.store.individual-id=true
+mosip.idp.cache.security.secretkey.reference-id=TRANSACTION_CACHE
+mosip.idp.cache.security.algorithm-name=AES/ECB/PKCS5Padding
+
+mosip.idp.cache.names=clientdetails,preauth,authenticated,authcodegenerated,userinfo,linkcodegenerated,linked,linkedcode,linkedauth,consented
+
+#spring.cache.type=redis
+#spring.cache.cache-names=${mosip.idp.cache.names}
+#spring.redis.host=localhost
+#spring.redis.port=6379
+management.health.redis.enabled=false
+
+spring.cache.type=simple
+mosip.idp.cache.key.hash.algorithm=SHA3-256
+mosip.idp.cache.size={'clientdetails' : 200, 'preauth': 200, 'authenticated': 200, 'authcodegenerated': 200, 'userinfo': 200, \
+   'linkcodegenerated' : 500, 'linked': 200 , 'linkedcode': 200, 'linkedauth' : 200 , 'consented' :200 }
+mosip.idp.cache.expire-in-seconds={'clientdetails' : 86400, 'preauth': 300, 'authenticated': 300, 'authcodegenerated': 300, \
+  'userinfo': ${mosip.idp.access-token-expire-seconds}, 'linkcodegenerated' : ${mosip.idp.link-code-expire-in-secs}, \
+  'linked': 300 , 'linkedcode': ${mosip.idp.link-code-expire-in-secs}, 'linkedauth' : 300, 'consented': 300 }
+
+## ------------------------------------------ Auth Wrapper ------------------------------------------------
+
+mosip.idp.authn.wrapper.impl=IdentityAuthenticationService
+mosip.idp.authn.ida.cert-url=https://raw.githubusercontent.com/mosip/mosip-config/qa-1201-B2/ida-partner.cer
```

Comment:
This should be changed to file-server URL

Reply:
`https://${mosip.api.host}/mosip-certs/ida-partner.cer`